dispatch/static/manager/src/js/components/inputs/ImageInput.js

```diff
@@ -24,6 +25,10 @@ function Image(props) {
 
 class ImageInputComponent extends React.Component {
 
+  componentDidMount() {
+    this.getSelected().map(id => this.props.getImage(this.props.token, id))
```

Comment:
I just realized that this could be a bit cleaner if it was moved to its own method:

```js
loadSelectedImages() {
  this.getSelected().map(id => this.props.getImage(this.props.token, id))
}
```

@Axel-Jacobsen @joncppl can one of you make this change for me?